src/pages/specification/specification-respec.html

```diff
@@ -117,12 +117,9 @@ <h3>
           Checking if Web Monetization is supported
         </h3>
         <p>
-          There are multiple ways to check if Web Monetization is supported.
```

Comment:
iirc ?? this file is generated, so you'll want to edit the base file, and then run the respec.sh command

Reply:
nevermind, this /IS/ the source
https://github.com/WICG/webmonetization/blob/main/src/pages/specification/respec.sh#L21